force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -1,368 +1,480 @@
-     * @param inputString The string to be repeated.
-     * @param repeatCount The number of times the string should be repeated.
-     * @return The new string with the inputString repeated repeatCount times.
-     */
-    public static String repeatString(String inputString, Integer repeatCount) {
-        // Initialize the result String
-
-        // Use a for loop to append the inputString to the result string repeatCount times
-        /*
-        for () {
-
-            // If it is not the last iteration, add a semicolon to separate the strings
-        }
-        */
-
-        // Return the final result string
-        return null; // Replace null with the variable you used to store the result
+    return result; // Replace null with the variable you used to store the result
+  }
+
+  /**
+   * Question 10
+   * Generate a string where the input string is repeated the given number of times, with each
+   * repetition separated by a semicolon.
+   * Do not add a semicolon after the last repetition.
+   * Example: repeatString("Hello World!", 3) should return "Hello World!; Hello World!; Hello World!"
+   * @param inputString The string to be repeated.
+   * @param repeatCount The number of times the string should be repeated.
+   * @return The new string with the inputString repeated repeatCount times.
+   */
+  public static String repeatString(String inputString, Integer repeatCount) {
+    // Initialize the result String
+    String result = '';
+    // Use a for loop to append the inputString to the result string repeatCount times
+    for (Integer i = 0; i < repeatCount; i++) {
+      result += inputString;
+      if (i < repeatCount - 1) {
+        result += '; ';
+      }
     }
 
-    /**
-     * Question 11
-     * Creates a List of integers and adds the integers 1, 2, and 3 to it.
-     * Example: createAndPopulateList() should return [1, 2, 3]
-     * @return A List of integers containing the numbers 1, 2, and 3.
-     */
-    public static List<Integer> createAndPopulateList() {
-        // Create a new list of integers
-
-        // Add the integers 1, 2, and 3 to the list
-        // add 1
-        // add 2
-        // add 3
-
-        // Return the populated list
-        return null; // Replace null with the variable you used to store the result
+    // Return the final result string
+    return result; // Replace null with the variable you used to store the result
+  }
+
+  /**
+   * Question 11
+   * Creates a List of integers and adds the integers 1, 2, and 3 to it.
+   * Example: createAndPopulateList() should return [1, 2, 3]
+   * @return A List of integers containing the numbers 1, 2, and 3.
+   */
+  public static List<Integer> createAndPopulateList() {
+    // Create a new list of integers
+    List<Integer> numberList = new List<Integer>();
+
+    // Add the integers 1, 2, and 3 to the list
+    // add 1
+    // add 2
+    // add 3
+    numberList.add(1);
+    numberList.add(2);
+    numberList.add(3);
```

Comment:
Try adding the numbers in the constructor. Instead of after the list is created. https://salesforce.stackexchange.com/questions/203694/how-to-initialize-list-set-values-by-construction-in-apex